ERROR - 2024-01-24 15:29:10 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'pos_base' in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos_base', 3306, '', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'pos_base' in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos-php\app\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Unidades->index()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-01-24 15:29:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'pos_base'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 15:35:37 --> mysqli_sql_exception: Table 'pos_base.unidades' doesn't exist in engine in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-24 15:35:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:35:37 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:35:37 --> [Caused by] mysqli_sql_exception: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `unidades`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 15:38:14 --> mysqli_sql_exception: Table 'pos_base.unidades' doesn't exist in engine in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-24 15:38:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:14 --> [Caused by] mysqli_sql_exception: Table 'pos_base.unidades' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `unidades`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `unidades`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Unidades.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Unidades->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 15:38:22 --> mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-24 15:38:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:22 --> [Caused by] mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `categorias`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 15:38:45 --> mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-24 15:38:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:45 --> [Caused by] mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `categorias`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 15:38:52 --> mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine in C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\pos-php\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\pos-php\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\pos-php\system\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos-php\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos-php\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos-php\app\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(941): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos-php\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos-php\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-01-24 15:38:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-24 15:38:52 --> [Caused by] mysqli_sql_exception: Table 'pos_base.categorias' doesn't exist in engine
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `categorias`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `categorias`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1613): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Categorias.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Categorias->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
